tests/sdk/queries/fileevents/filters/test_device_filter.py

```diff
@@ -173,3 +174,9 @@ def test_public_ip_address_not_in_str_gives_correct_json_representation():
     _filter = PublicIPAddress.not_in(items)
     expected = NOT_IN.format("publicIpAddress", *items)
     assert str(_filter) == expected
+
+
+def test_device_signed_in_username_gives_correct_json_representation():
```

Comment:
When we write this tests, we test all of the valid operations, not just `eq`. Take a look at the other tests / filters tested in this file for examples. It should be pretty easy to essentially copy/paste them and change the appropriate fields.

Reply:
Added more tests.